main.go

```diff
@@ -117,6 +117,9 @@ func main() {
 	// Service Orchestration handlers
 	api.HandleFunc("/{account}/services", ServiceOrchestrationCreateHandler).Methods(http.MethodPost)
 	api.HandleFunc("/{account}/services", ServiceOrchestrationDeleteHandler).Methods(http.MethodDelete)
+	api.HandleFunc("/{account}/services", func(w http.ResponseWriter, req *http.Request) {
+		w.WriteHeader(http.StatusNotImplemented)
```

Comment:
you might still want to do the stuff to check if the account exists?

Reply:
ok, i created a handler for that endpoint